src/setup_node_env/node_version_validator.js

```diff
@@ -37,10 +37,11 @@ var rawRequiredVersion = (pkg && pkg.engines && pkg.engines.node) || null;
 var requiredVersion = rawRequiredVersion ? 'v' + rawRequiredVersion : rawRequiredVersion;
 var currentVersionMajorMinorPatch = currentVersion.match(/^v(\d+)\.(\d+)\.(\d+)/);
 var requiredVersionMajorMinorPatch = requiredVersion.match(/^v(\d+)\.(\d+)\.(\d+)/);
+var currentMinorPatch =  currentVersionMajorMinorPatch[2].concat(".", currentVersionMajorMinorPatch[3]);
+var requiredMinorPatch = requiredVersionMajorMinorPatch[2].concat(".", requiredVersionMajorMinorPatch[3]);
 var isVersionValid =
   currentVersionMajorMinorPatch[1] === requiredVersionMajorMinorPatch[1] &&
-  currentVersionMajorMinorPatch[2] === requiredVersionMajorMinorPatch[2] &&
-  parseInt(currentVersionMajorMinorPatch[3], 10) >= parseInt(requiredVersionMajorMinorPatch[3], 10);
+  parseFloat(currentMinorPatch) >= parseFloat(requiredMinorPatch);
 
 // Validates current the NodeJS version compatibility when OpenSearch Dashboards starts.
 if (!isVersionValid) {
```

Comment:
The error message bellow needs to be updated accordingly: a higher minor version is fine too.

Reply:
I added warning message.

Reply:
> The error message bellow needs to be updated accordingly: a higher minor version is fine too.

Got it makes sense